### PR TITLE
`Development`: Add LTI 1.3 documentation for Moodle admin

### PR DESCRIPTION
**Pull request by basak-akan**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

Rearrange the current documentation page for Moodle admins and instructors to provide a more clear guide. 

### Description

Lti configuration steps for instructors and moodle admins are separated. New warning added at the beginning of the steps for how to configure exercises which indicates that lti configuration needs to be successfully completed as a prerequisite. 

### Steps for Testing
Use the following link to check the documentation: https://artemis-platform--7448.org.readthedocs.build/en/7448/user/lti.html 
Please go through the documentation and check if it is understandable.

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### CODE REVIEW

File: docs/user/lti.rst [hunk 1]
```diff
@@ -84,7 +84,7 @@ With the Online Course setting enabled, you can now proceed to integrate Artemis
 
 1.2. Copy Dynamic Registration URL from Artemis
 """""""""""""""""""""""""""""""""""""""""""""""
-Once the above setting is enabled, you can now set up a bridge between Artemis and Moodle.
+Once the above setting is enabled, you can now request your Moodle Admin to set up a bridge between Artemis and Moodle. Moodle Admin needs course Dynamic Registration URL to successfully complete the configuration.
```
Comment (jakubriegel) [inline docs/user/lti.rst:87]: _Moodle Admin needs **the** course Dynamic Registration_

**Review by jakubriegel — APPROVED** [review]
Docs look ok

File: docs/user/lti.rst (hunk 1 above)
Comment (basak-akan) [reply docs/user/lti.rst]: Thank you for noticing I added your suggestion 👍

**Review by egekurt123 — APPROVED** [review]
Documentation text and instructions are clear and easy readable

**Review by JanaNF — CHANGES_REQUESTED** [review]
The documentation looks fine but when I started to copy it into Grammarly some spelling mistakes (e.g. missing commas/ words) were shown. I would suggest copying your text into Grammarly yourself and see if the noted changes are something you want to integrate (because of the TUM we get free access to the education plan).

**Comment by dearjasmina** [comment]
> The documentation looks fine and the instructions are clear but when I started to copy it into Grammarly some spelling mistakes (e.g. missing commas/ words) were shown. I would suggest copying your text into Grammarly yourself and see if the noted changes are something you want to integrate (because of the TUM we get free access to the education plan).

Also, please consider removing passive usage from the text

**Review by JanaNF — APPROVED** [review]

**Review by RY997 — APPROVED** [review]
Looks good to me

**Pull request merged**